src/panels/lovelace/cards/hui-entities-card.ts

```diff
@@ -148,23 +140,11 @@ class HuiEntitiesCard extends LitElement implements LovelaceCard {
     if (this._hass) {
       element.hass = this._hass;
     }
-    if (
-      entityConf.entity &&
-      !DOMAINS_HIDE_MORE_INFO.includes(computeDomain(entityConf.entity))
-    ) {
-      element.classList.add("state-card-dialog");
-      element.addEventListener("click", () => this._handleClick(entityConf));
```

Comment:
We now changed the behaviour of all entity rows? You can no longer click on the row to get the `more-info`?

Reply:
moved actions to the badge